k8s/demo/common/ccd/ccd-full.yaml

```diff
@@ -6,7 +6,19 @@ metadata:
   namespace: ccd
   annotations:
     flux.weave.works/automated: 'true'
-    flux.weave.works/tag.java: 'glob:prod-*'
+    flux.weave.works/tag.ccd-full-admin-web: glob:prod-*
+    flux.weave.works/tag.ccd-full-api-gateway: glob:prod-*
+    flux.weave.works/tag.ccd-full-case-management-web: glob:prod-*
```

Comment:
bsp has slightly different config without release-name prefix. I'm not sure which one is good practice.